_posts/events/2017-11-14-mumttt.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+categories: [event]
+event_name: "The 3rd Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2017)"
+event_url: "http://rgcl.wlv.ac.uk/europhras2017/mumttt-2017/"
+event_location: "London, UK"
+event_submission_date: "2017-06-15"
+host_name: "europhras 2017"
+host_url: "http://rgcl.wlv.ac.uk/europhras2017/"
+siglex_endorsed: yes
+---
+The 3rd Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2017)
+
+London, UK, 13-14 November 2017
```

Comment:
You probably want two spaces at the end of this line so that the URL shows up on its own line.

Reply:
Done